activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -132,6 +132,7 @@ def underscore(camel_cased_word)
     #
     def humanize(lower_case_and_underscored_word, capitalize: true, keep_id_suffix: false)
       result = lower_case_and_underscored_word.to_s.dup
+      return result if result.empty?
```

Comment:
I guess the question here is how common can it be to call this on an empty string, and whether that `empty?` call overhead justify itself even if it's very cheap.

Reply:
Yeah, but I had to add this because I saw a test failure in this Action View helper test. https://github.com/rails/rails/blob/d7cfaeb1528abefdc6f27c21feb748ad715182c9/actionview/test/template/form_tag_helper_test.rb#L646-L648
Perhaps we could update that helper not to call this method with an empty string, but anyway it'd be better to have a guard for such case.

Reply:
Interesting. LGTM then 👍